packages/framework/src/Resources/views/Common/Inline/Icon/icon.html.twig

```diff
@@ -0,0 +1,8 @@
+{% if type == 'svg-font' %}
+    <i class="svg svg-{{ name }} {{ attr.class|default('')}}"
```

Comment:
I am wondering why the `attr.class` is string. IMHO the list will do much better and will be more intuitive to use.

Reply:
Because class is one of attributes and all attributes are string. Even in forms you set attributes as strings.